transaction.go

```diff
@@ -0,0 +1,158 @@
+package seth
+
+import (
+	"bytes"
+	"math/big"
+)
+
+type Encoding int
+
+const (
+	LIST Encoding = iota
```

Comment:
These will be exported as top-level identifiers, which will muddy the output of `go doc`. (Also, the canonical Go style avoids casing constants any differently than regular identifiers.)